src/app/containers/MostRead/index.jsx

```diff
@@ -25,18 +25,29 @@ const MostReadContainer = ({ endpointOverride }) => {
     return null;
   }
 
-  const endpoint =
-    endpointOverride || getMostReadEndpoint({ service, variant });
+  const isTestOrLocal =
+    process.env.SIMORGH_APP_ENV === 'test' ||
+    process.env.SIMORGH_APP_ENV === 'local';
 
-  return <Canonical endpoint={endpoint} />;
+  const endpoint =
+    mostReadEndpointOverride || getMostReadEndpoint({ service, variant });
+
+  return (
+    <Canonical
+      endpoint={endpoint}
+      forceMostRead={forceMostRead || isTestOrLocal}
```

Comment:
Better to pull `forceMostRead || isTestOrLocal` out into a const above and add an explanatory comment as to why we're doing this. 

Note that this appears that we will enable us to show mostread on the Simorgh Live env if `forceMostRead` prop passed down is true. Are we sure we want to allow this?

Reply:
> Note that this appears that we will enable us to show mostread on the Simorgh Live env if forceMostRead prop passed down is true. Are we sure we want to allow this?

because this prop is explicitly for storybook, would it better if i rename the prop. i.e mostReadOnStorybook

Reply:
If we need to render most read within a front page in Storybook using fixture data, could we use time machine to fix the time in the storybook story instead of allowing us to force it to display?

Reply:
>If we need to render most read within a front page in Storybook using fixture data, could we use time machine to fix the time in the storybook story instead of allowing us to force it to display?

this just gave me the thought of changing the fixture data, so that it's 1000s of years in the future, which will have the same effect.

Reply:
https://github.com/bbc/simorgh/blob/latest/.storybook/time-machine.js#L7-L13

I don't wanna mess around with what's set for timemachine already